src/stories/NewModal.story.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <Story title="New Modal (HoppModal)">
+    <Variant title="Simple Dialog with Close Button">
+      <button @click="openTestModal">Open Simple Dialog</button>
+    </Variant>
+    <Variant title="Dialog with Input">
+      <p>
+        <span>
+          Resolved Value: <strong>{{ text }}</strong>
+        </span>
+      </p>
+
+      <button @click="openInputDialog">Open Dialog with Input</button>
+    </Variant>
+  </Story>
+</template>
+
+<script lang="ts" setup>
+import { ref } from "vue"
+import { useModals } from "vue-promise-modals"
+import { HoppModal, InputDialog } from "../components/modal"
+
+const { openModal } = useModals()
+
+function openTestModal() {
+  openModal(HoppModal, {
+    openCount: 0,
```

Comment:
Here, `openCount` is given a static value which won't update. We can give it an incrementing variable so that it shows the number of times the modal is opened.